crates/source_analyzer/src/cyclers.rs

```diff
@@ -0,0 +1,225 @@
+use std::{
+    collections::{HashMap, HashSet},
+    path::Path,
+};
+
+use quote::format_ident;
+use serde::Deserialize;
+use syn::Ident;
+use topological_sort::TopologicalSort;
+
+use crate::{
+    contexts::Field,
+    error::Error,
+    manifest::{CyclerManifest, FrameworkManifest},
+    node::Node,
+};
+
+pub type CyclerName = Ident;
+pub type InstanceName = Ident;
+pub type OutputName = Ident;
+
+#[derive(Debug)]
+pub struct Instance {
+    pub name: InstanceName,
+}
```

Comment:
Remove struct as discussed in a previous discussion.

Reply:
Done